frontend/src/components/OntologyTable.tsx

```diff
@@ -43,11 +45,17 @@ const renderOntology = (
 const OntologyTable: React.FC = () => {
   const [selectedNode, setSelectedNode] = useState(initialNode);
   const [ontologies, setOntologies] = useState([]);
+  const [, setApiError] = useRecoilState(apiErrorState);
```

Comment:
When you don't need the value (just the setter function) you can use useSetRecoilState instead. There's also useRecoilValue when you only need the value and not the setter.